### DISCUSSION TITLE
Tracking - almost correct but still not

### DISCUSSION THREAD

**Discussion by canonex** [alicevision/Meshroom#2309] [Q&A]
Hi!

I made a video, extracted the frames (not with Meshroom, I'm one with trouble #1291), started the tracking and photogrammetry pipeline. After many hours (670 frames) I finally got the result!

I imported the Alembic camera, imported the obj: the tracking is generally correct, but the minimal shaking and vibrations of the camera are missing.

I have attached the mg file (renamed txt to allow the attach here on Github) and the result.
https://github.com/alicevision/Meshroom/assets/1444378/a10a9208-c422-49cf-88a7-158491d0e3b5
[MeshRoomTrack.txt](https://github.com/alicevision/Meshroom/files/14102556/MeshRoomTrack.txt)

How can I improve the result?
_And here's a side question: is it possible to speed up the process? CCTag can be helpful or faster?_

Thank you!
Riccardo

**Comment by natowi** [comment]
With 670 Frames you are probably skipping a few frames that would introduce the shaking?
Really nice tracking result btw.

**Reply by canonex** [reply]
What do you mean by "skipping"?

I'm sure all the frames are there and are loaded in Meshroom!
Could there be some rounding parameter that skips some frames?

Thank you,
Riccardo

**Comment by canonex** [comment]
The video is a little more than 22 seconds, exactly 670 frames at 30 fps: here it is!
https://we.tl/t-El8BLdKFXF
(54Mb. Sorry, can't load here, it is too big, max is 10Mb)

I checked the MeshRoomTrack.mg ( https://github.com/alicevision/Meshroom/assets/1444378/a10a9208-c422-49cf-88a7-158491d0e3b5
[MeshRoomTrack.txt](https://github.com/alicevision/Meshroom/files/14102556/MeshRoomTrack.txt) ) and all the 670 frames are correctly loaded.

The frames where exported using ffmpeg `(like ffmpeg -r 1 -i "$1" -r 1 $filename%03d.png)`. 

Thank you,
Riccardo
